src/api/spec/components/notification_excerpt_component_spec.rb

```diff
@@ -2,21 +2,31 @@
 
 RSpec.describe NotificationExcerptComponent, type: :component do
   let(:user) { create(:user) }
-  let(:notification_for_projects_comment) { create(:web_notification, :comment_for_project, notifiable: comment, subscriber: user) }
 
-  context 'with short excerpt' do
+  context 'notification for a BsRequest without a description' do
+    let(:bs_request) { create(:bs_request_with_submit_action, description: nil) }
+    let(:notification) { create(:web_notification, :request_created, notifiable: bs_request, subscriber: user) }
+
+    it do
+      expect(render_inline(described_class.new(notification))).not_to have_text('...')
```

Comment:
This is checking the string for not having the text `...`. With a description with a `nil` value, shouldn't we check for the empty string?

Reply:
Good point, I changed the specs to check for an empty string.